test/bats/tests/vault/vault_synck8s_v1alpha1_secretproviderclass.yaml

```diff
@@ -28,3 +30,7 @@ spec:
           objectPath: "v1/secret/foo1"
           objectName: "bar1"
           objectVersion: ""
+        - |
+          objectPath: "v1/secret/foo1"
+          objectName: "nest/foo1"
```

Comment:
For `0.0.7` version of the provider, objectName is used to select the 'key' within the `v1/secret/foo1` secret and re-uses that as the file path.

To do the desired test i'll need to seed a new secret (or include a nested path in a key of an existing seeded secret).